apps/etterlatte-saksbehandling-ui/client/src/components/behandling/aktivitetsplikt/AktivitetspliktTidslinje.tsx

```diff
@@ -0,0 +1,150 @@
+import { Buildings2Icon, HatSchoolIcon, PencilIcon, PersonIcon, RulerIcon } from '@navikt/aksel-icons'
+import { Alert, Timeline } from '@navikt/ds-react'
+import { hentAktiviteter } from '~shared/api/aktivitetsplikt'
+import { formaterDato, formaterDatoMedTidspunkt } from '~utils/formattering'
+import { IDetaljertBehandling } from '~shared/types/IDetaljertBehandling'
+import { addMonths, addYears } from 'date-fns'
+import { useApiCall } from '~shared/hooks/useApiCall'
+import React, { useEffect, useState } from 'react'
+import { AktivitetspliktType, IAktivitet } from '~shared/types/Aktivitetsplikt'
+import { isFailureHandler } from '~shared/api/IsFailureHandler'
+import { NyAktivitet } from '~components/behandling/aktivitetsplikt/NyAktivitet'
+import styled from 'styled-components'
+
+export const AktivitetspliktTidslinje = (props: { behandling: IDetaljertBehandling; doedsdato: Date }) => {
+  const { behandling, doedsdato } = props
+  const [hentet, hent] = useApiCall(hentAktiviteter)
+  const seksMndEtterDoedsfall = addMonths(doedsdato, 6)
+  const tolvMndEtterDoedsfall = addMonths(doedsdato, 12)
+
+  const [aktiviteter, setAktiviteter] = useState<IAktivitet[]>([])
+  const [aktivitetsTypeProps, setAktivitetsTypeProps] = useState<AktivitetstypeProps[]>([])
+
+  useEffect(() => {
+    hent({ behandlingId: behandling.id }, (aktiviteter) => {
+      oppdaterAktiviteter(aktiviteter)
+    })
+  }, [])
+
+  const oppdaterAktiviteter = (aktiviteter: IAktivitet[]) => {
+    setAktivitetsTypeProps([...new Set(aktiviteter.map((a) => a.type))].map(mapAktivitetstypeProps))
+    setAktiviteter(aktiviteter)
+  }
+
+  return (
+    <TidslinjeWrapper className="min-w-[800px]">
+      {aktiviteter.length === 0 ? (
+        <Alert variant="info" inline>
+          Ingen aktiviteter er registrert.
+        </Alert>
+      ) : (
+        <Timeline startDate={doedsdato} endDate={addYears(doedsdato, 3)}>
+          <Timeline.Pin date={doedsdato}>
+            <p>Dødsdato: {formaterDato(doedsdato)}</p>
+          </Timeline.Pin>
+          <Timeline.Pin date={new Date()}>
```

Comment:
Til senere: Skulle man vurdert å ha en egen farge på pin for dagens dato for å skille den fra viktige hendelser?

Reply:
Høres ut som en god idé. Det er et par andre småting som bør justeres litt også. F. eks at den Pin-en detter lengre nedover første rad jo flere rader man har 🤷‍♂️ .